main/CMakeLists.txt

```diff
@@ -143,7 +140,7 @@ if (MINGW)
             DESTINATION bin)
       endif (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.12.4")
       install( FILES
-         ${DEPENDENCIES_DIR}/portaudio.dll RENAME libportaudio-x86_64-w64-mingw32.static.dll
+         ${DEPENDENCIES_DIR}/portaudio.dll
```

Comment:
This one is part of #5457

Reply:
If and when that PR is merged, I can easily rebase and solve the conflict.

Reply:
Like now ;-)

Reply:
Rebase done :-)